client/a8c-for-agencies/components/a4a-confirmation-dialog/style.scss

```
@@ -7,6 +7,15 @@
 		@include heading-large;
 	}
 
+	& .dialog__content {
+		h1.a4a-confirmation-dialog__heading {
+			font-size: rem(24px);
+		}
+		p, div {
+			font-size: rem(14px);
```

Since we don't have an equivalent for 14px, we could use 13px pr 15px:

```suggestion
			@include body-medium; // Or @include body-large if it is too small
```